src/partials/footer.html

```diff
@@ -0,0 +1,29 @@
+<footer class="footer">
+  <div class="footer__container">
+    <div class="footer__logo">
+      <svg class="footer__icon">
+        <use href="./images/sprite.svg#icon-logo"></use>
+      </svg>
+      <div class="footer__tablet">
+        <div class="footer__logo--tel">
```

Comment:
Modyfikator w BEM nie powinien być stosowany bez klasy głównej. Jeżeli użyty jest footer__logo--tel to powinien być również użyty footer__logo. Proponuję użycie tutaj osobnej klasy